docs/understand_mmcv/io.md

```diff
@@ -2,11 +2,17 @@
 
 This module provides two universal API to load and dump files of different formats.
 
+```{note}
+In v1.3.15 and later, `File IO` also supports loading data from different backends and dumping data to different backends. More datails at https://github.com/open-mmlab/mmcv/pull/1330.
```

Comment:
Since v1.3.15, the IO modules support loading and dumping data from and to different backends, respectively. More details are in `PR #1330 https://github.com/open-mmlab/mmcv/pull/1330`_.